packages/interaction/src/InteractionManager.js

```diff
@@ -1065,7 +1089,7 @@ export default class InteractionManager extends EventEmitter
                 const child = children[i];
 
                 // time to get recursive.. if this function will return if something is hit..
-                const childHit = this.processInteractive(interactionEvent, child, func, hitTest, interactiveParent);
+                const childHit = this.processInteractive(interactionEvent, child, func, hitTest, interactiveParent, true);
```

Comment:
For recursive-sake, shouldn't this value be passed along here?

```suggestion
                const childHit = this.processInteractive(interactionEvent, child, func, hitTest, interactiveParent, delayed);
```